Comments design

dartcafe opened the pull request.

splitt3r commented on js/vote.js:
```diff
@@ -144,8 +142,18 @@ $(document).ready(function () {
-		$.post(form.action, data, function (data) {
-			$('.comments .comment:first').after('<div class="comment"><div class="comment-header"><span class="comment-date">' + data.date + '</span>' + data.userName + '</div><div class="wordwrap comment-content">' + data.comment + '</div></div>');
+		$.post(form.action, data, function(data) {
+		var newCommentElement = '<li class="comment column"> ' +
```
We should use [Handlebars ](https://docs.nextcloud.com/server/12/developer_manual/app/tutorial.html?highlight=handlebar#creating-a-navigation) here.

dartcafe replied:
Awesome. Was the missing link for me. I will go into it later. 👍

dartcafe commented:
Little preview.
![screenshot-2017-11-11 umfragen - polls](https://user-images.githubusercontent.com/26707476/32690594-72ccd2ee-c6f9-11e7-94ab-5eb4de10f3a3.png)

v1r0x commented:
Looks good 👍 Will have closer look at it the next days.

splitt3r commented:
👍 the textbox should have the same size and placeholder as the files app one. And perhaps we should make the upper part of the sidebar look more streamlined to the official nextcloud sidebars without the stroke + icons with description as tooltip?

dartcafe commented:
![grafik](https://user-images.githubusercontent.com/26707476/32696928-997dd984-c784-11e7-84da-b04be7ef4aa1.png)
![screenshot-2017-11-12 umfragen - polls](https://user-images.githubusercontent.com/26707476/32696930-a0a56e7a-c784-11e7-9d6a-1970ee458e7c.png)

dartcafe commented:
????

![grafik](https://user-images.githubusercontent.com/26707476/32745222-5c9e6c0c-c8b2-11e7-88a9-b8029f50f312.png)
![grafik](https://user-images.githubusercontent.com/26707476/32745264-8201d6be-c8b2-11e7-8cbc-995d2621832c.png)

Forced reload?

splitt3r commented:
Two very small points 😁 how about a bit more space between owner and tags? And a placeholder like 'New comment...' would be great.

v1r0x commented:
@dartcafe Yes, I always have the dev console open and http cache disabled. But even Ctrl+F5 doesn't help. Still no box.

@splitt3r The placeholder exists, but not visible, because the input field is a div, not a `<textarea>`. Any reason for that @dartcafe ?

dartcafe commented:
- Added placeholder
- added some margin to the tags

> The placeholder exists, but not visible, because the input field is a div, not a <textarea>. Any reason for that ?

I just adopted the comments app. The div is nicer, because it automatically resizes depending of the content.

Please check https://nc13.yagst.de/index.php/apps/polls/poll/12rdzh9QYiFZaFz4 
I cannot reproduce the issue of the missing text area.

v1r0x commented:
I pulled the latest commits and now I see the placeholder and can type a comment. But the border is missing.

![screenshot-2017-11-15 umfragen - nextcloud](https://user-images.githubusercontent.com/9248714/32827070-27a9c3a4-c9eb-11e7-8b80-c7b4887175d9.png)

dartcafe commented:
Aiaiai. Got it. I am checking against NC13. In NC12 this issue occures. 

Please fill out the support form next time. :-) I'll gonna fix this.

v1r0x commented:
Is there a support form for comments? :wink: 
I'm always on latest stable release. Needs testing as well :stuck_out_tongue:

splitt3r commented:
Will have a look at it :) If you want to have a look at getting started with vue + nc you could have a look at my [repo](https://github.com/splitt3r/mindmaps/tree/feature/rework-js) or [ChristophWursts](https://github.com/ChristophWurst/nextcloud_vue). It´s far away from ready but got the basic stuff working.

splitt3r commented:
The home icon seems to be wrong positioned.

![Error](https://user-images.githubusercontent.com/1869584/33095734-fc19758a-cf04-11e7-930a-0c976f13d5f5.png)

dartcafe commented:
@splitt3r Sure, you pulled the current status of the PR? Your screen shot display the old layout. See my first comments for the new design of the comments...
The problem with the positioning should be fixed (occured in Chrome and Safari).

splitt3r commented:
Will check that again 😁

v1r0x commented:
I can see a border! 😁 
One last thing from my side: The cursor doesn't switch to `text`, so it's confusing whether it's a text field or not.

dartcafe commented:
@splitt3r I don't have this issue neither in oc10, nc11, nc12 nor nc13. ❓ 
I added a border to the exoiry info. 
@v1r0x Which text field?
![grafik](https://user-images.githubusercontent.com/26707476/33227709-cf5ab638-d1a9-11e7-8f2a-ec6ad90e782d.png)

dartcafe commented:
@splitt3r Surprise, surprise. 
Just pulled the latest master of the daily channel (13.0.0 Beta 1), and I have this issue too. Will check that later.

dartcafe commented:
Fixed the layout for NC13. Still issues in oc10-nc12. Fix it later.

v1r0x commented:
> @v1r0x Which text field?

The one on the screenshot. For me (on nc12) there is the default cursor, not the text cursor 😕

dartcafe commented:
Please review. Fixed the controls (breadcrump) and the cursor for the text box. For me, I am ready to merge.

splitt3r commented:
One small thing left here (Nc 13):

![Icon](https://user-images.githubusercontent.com/1869584/33241391-de8d7340-d2c4-11e7-8a95-14720fca882e.png)

The icon is left aligned and the number isn´t centered anymore 😁

dartcafe commented:
@splitt3r Travis is not good to me. :-)  And where is scrutinizer?
BTW @splitt3r and @v1r0x, if you could just close the CRs???

splitt3r commented:
:joy: disk quota exceeded will habe a look at the Travis builds. What do you mean with CRs?

dartcafe commented:
CR = Change Request.
